_sass/footer.scss

```diff
@@ -6,15 +6,15 @@ footer {
 footer span.copyright {
   text-transform: uppercase;
   text-transform: none;
-  font-family: $title-font;
+  font-family: sans-serif, $title-font;
```

Comment:
why is standard sans-serif first?

Reply:
The custom font displays the copyright symbol weird, and also moves the social media symbols too far up; so I changed the footer font to sans-serif.

Reply:
ah, I see, could you just remove the `, $title-font` then, that will never render if `sans-serif` is first

Reply:
ok